common/init

```diff
@@ -24,4 +24,6 @@ else
 fi
 
 export SNAP_LAUNCHER_ARCH_TRIPLET=$ARCH
-
+if [ -f $SNAP/bindtextdomain.so ]; then
+  export LD_PRELOAD=$SNAP/bindtextdomain.so
```

Comment:
If an app wanted to provide their own set of overridden functions, shouldn't this be:

    export LD_PRELOAD=$LD_PRELOAD:$SNAP/bindtextdomain.so

Not sure whether that would work?

Also you might want to keep the blank line at the end of the file to make the generated launcher more readable.

Reply:
That does indeed work, I've updated to include that as well as adding the empty line to the end